strawberryfields/gbs/data.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+GBS Datasets
+============
+
+**Module name:** :mod:`strawberryfields.gbs.data`
+
+.. currentmodule:: strawberryfields.gbs.data
+
+This module provides access to pre-calculated datasets of simulated GBS samples generated from a
+range of input graphs. Each dataset corresponds to a set of samples generated from a fixed graph
+with corresponding values for:

```

Comment:
Would it make sense to have a `description` attribute that gives more information about the dataset? For example, playing around with the `Planted` dataset I didn't find information regarding which nodes correspond to the planted subgraph. For other datasets we may want to give similar information, like location of max clique, or what class the MUTAG graphs belong to

Reply:
I think that's a good approach; it can then be shown via `help(Planted)`. Similarly to how it works with NumPy.

Reply:
Great!

I added:
"The 10-node planted clique is contained within the final 10 nodes of the graph." to the `Planted` docstring